yvalenta/spec/connection_spec.rb

```diff
@@ -0,0 +1,78 @@
+require "spec_helper"
+
+describe 'Connection' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.